Projects/Team3/OpencartTestFramework/Tests/TestsAPI/GetCartInfoAPI.cs

```diff
@@ -0,0 +1,19 @@
+﻿using NUnit.Framework;
+using RestSharp;
+using System;
+
+namespace OpencartTestFramework.Tests.TestsAPI
+{
+    public class GetCartInfoAPI:RestTests
+    {
+        [Test]
+        public void GetInfo()
+        {
+            var client = new RestClient("http://127.0.0.1/opencart/index.php?route=api/cart/products/&api_token=ff7e87b179375856d3c691872a");
```

Comment:
Move URL to constants.
Do not use a hardcoded token.
you need to generate a read-only field